app/components/Input.tsx

```
@@ -0,0 +1,193 @@
+                            const colorName = getCategoryColorName(graph.CategoriesMap.get(label)!.index)
+                            const color = getCategoryColorValue(graph.CategoriesMap.get(label)!.index)
+                            return (
+                                <button
+                                    className={cn(
+                                        "w-full flex gap-3 p-1 items-center rounded-md",
+                                        selectedOption === index && "bg-gray-100"
+                                    )}
+                                    onMouseEnter={() => setSelectedOption(index)}
+                                    onMouseLeave={() => setSelectedOption(-1)}
+                                    onClick={() => {
+                                        debugger
```

Biome

[error] This is an unexpected use of the debugger statement.

Unsafe fix: Remove debugger statement


(lint/suspicious/noDebugger)